src/tweet.js

```diff
@@ -138,7 +138,7 @@ export default class Tweet {
         this._updateData(tweet);
       })
       .catch((error) => {
-        if (this.logger) {this.logger.error(`Could not display tweets - ${error}`);}
+        if (this.logger) {this.logger.error(`Could not display tweet - ${JSON.stringify(tweet)}`);}
```

Comment:
@jungeunyoon why are you not printing the error object anymore, does it not give any useful info ?

Reply:
@santiagonoguez Fair point. I'll change it to include it as well. We've actually never gotten this error before so I couldn't tell you for sure. Just wanted to include the tweet itself. Thanks!